tests/Unit/PartnerSearchTest.php

```
@@ -0,0 +1,167 @@
+            ->willReturn($expectedPartners);
+
+        // Act
+        $partners = $appClient->searchPartners($peppolId);
+
+        // Assert
+        $this->assertIsArray($partners);
+        $this->assertNotEmpty($partners);
+        $this->assertCount(1, $partners);
+        $this->assertEquals('Example Partner Company', $partners[0]['name']);
+        $this->assertEquals($peppolId, $partners[0]['peppolId']);
+    }
```

PHPMD (2.15.0)

The method it_finds_partner_by_peppol_id is not named in camelCase. (undefined)

(CamelCaseMethodName)